src/server.h

```diff
@@ -2867,6 +2868,7 @@ void sortroCommand(client *c);
 void lremCommand(client *c);
 void lposCommand(client *c);
 void rpoplpushCommand(client *c);
+void lmMoveCommand(client *c);
```

Comment:
I think it is more appropriate to use `lmmoveCommand`, like the other command methods named.